src/estimagic/visualization/estimation_table.py

```diff
@@ -598,24 +1240,29 @@ def _generate_notes_latex(append_notes, notes_label, sig_levels, custom_notes, d
     """
     n_levels = df.index.nlevels
     n_columns = len(df.columns)
-    sig_levels = sorted(sig_levels)
+    significance_levels = sorted(significance_levels)
     notes_text = "\\midrule\n"
```

Comment:
This midrule is not needed if `append_notes == False`

Reply:
I suggest to move `\\midrule\n` in the if statement below

Reply:
Done.